Content.Server/GameObjects/Components/Mobs/Body/Organs/BrainOrgan.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using YamlDotNet.RepresentationModel;
+using Content.Server.Interfaces.Chat;
+using Robust.Shared.IoC;
+using Robust.Shared.Maths;
+using Robust.Shared.Utility;
+
+namespace Content.Server.GameObjects.Components.Mobs.Body.Organs
+{
+    /// <summary>
+    /// Handles everything associated with brain damage and mind-altering states
+    /// </summary>
+    public class Brain : Organ
+    {
+        IChatManager chat;
+        List<string> brainDamagePhrases;
+
+        public override void Startup()
+        {
+            base.Startup();
+            chat = IoCManager.Resolve<IChatManager>();
+        }
+
+        public override void ExposeData(YamlMappingNode mapping)
+        {
+            brainDamagePhrases = new List<string>();
+            foreach (var prot in mapping.GetNode<YamlSequenceNode>("brainDamageLines").Cast<YamlMappingNode>())
```

Comment:
You don't need to make this a sequence of mappings. Just lay the YAML out like this and pull strings from each element directly:
```yml
  brainDamageLines:
    - "HURR DURR IS SPESSMENS READY YET?"
    - "FUS RO DAH"
    - "fucking 4rries!"
    - "stat me"
    - ">my face" 
```